extensions/amp-story/1.0/amp-story-system-layer.css

```diff
@@ -82,15 +82,17 @@
   float: left !important;
 }
 
+ol.i-amphtml-animate-progress li {
+  transition: transform 1s cubic-bezier(0.4, 0.0, 0.2, 1) !important;
```

Comment:
We'd need to check the animation speed with Hong, but I feel like a beet faster, like `.8s`, could be an ok place to start